libraries/elixir/README.md

```diff
@@ -0,0 +1,58 @@
+Elixir library for Standard Webhooks
+
+# Example
+
+Verifying a webhook payload:
+
+```elixir
+StandardWebhooks.verify(webhook_payload, plug_conn, webhook_secret)
```

Comment:
@cpursley, not just the ordering, but also having an instance.

Here is how it looks like in Ruby (which you mentioned):
```ruby
wh = StandardWebhooks::Webhook.new(base64_secret)
wh.verify(webhook_payload, webhook_headers)
```

Is it possible to do it here too, or is it very much not how people do it in Elixir? That's what I was referring to in https://github.com/standard-webhooks/standard-webhooks/pull/87/#discussion_r1438942237

Reply:
Elixir doesn't have instances. It is a functional language without objects. It has structs like golang but you can't bind methods them.

Reply:
Gotcha, sorry for the noise. :)